ivadomed/main.py

```diff
@@ -430,7 +443,7 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
                                'training')
 
         # Choice of optimisation metric
-        metric = "recall_specificity" if model_params["name"] in imed_utils.CLASSIFIER_LIST else "dice"
+        metric = "recall_specificity" if model_params[ModelParamsKW.NAME] in imed_utils.CLASSIFIER_LIST else "dice"
```

Comment:
Could you also extract these two keywords? as the metrics keywords: `recall_specificity` and `dice`. There might be more.